miio/exceptions.py

```diff
@@ -6,3 +6,8 @@ class DeviceException(Exception):
 class DeviceError(DeviceException):
     """Exception communicating an error delivered by the target device."""
     pass
+
+
+class RecoverableError(DeviceException):
```

Comment:
Looks good to me in general, but maybe `RecoverableDeviceError` deriving from `DeviceError` instead?

Reply:
Fixed.